src/onnx/parse_resize.cpp

```diff
@@ -33,49 +33,25 @@ namespace migraphx {
 inline namespace MIGRAPHX_INLINE_NS {
 namespace onnx {
 
-static std::vector<int>
+static void
 calc_neighbor_points(const std::vector<std::vector<std::vector<std::size_t>>>& vvv_ind,
-                     int i_dim,
-                     std::vector<std::vector<std::size_t>> vec_dims,
-                     const shape& in_s)
+                     const int& n_dims,
```

Comment:
make ndims unsigned , if you bitshift later  it'll cause undefined behavior

Reply:
aha correct, it should be unsigned anyway.